include/osquery/query.h

```diff
@@ -487,7 +485,7 @@ class Query {
    *
    * @return the success or failure of the operation.
    */
-  Status addNewResults(const QueryData& qd,
+  Status addNewResults(QueryData qd,
```

Comment:
Why did you make this change?

Reply:
I also made required change -
status = dbQuery.addNewResults(std::move(sql.rows()), item.epoch, item.counter, diff_results);

Since inside 'addNewResults' the intention is to move sql.rows() 
136   } else {
137     dr.added = std::move(current_qd); <--
138     target_gd = &dr.added;
139   }

with pass by const reference statement -
dr.added = std::move(current_qd); 
calls copy assignment 

With this change it calls move assignment.